src/libs/histogram.c

```diff
@@ -584,13 +584,15 @@ static gboolean _lib_histogram_motion_notify_callback(GtkWidget *widget, GdkEven
     else if(x > d->green_x && x < d->green_x + d->button_w && y > d->button_y && y < d->button_y + d->button_h)
     {
       d->highlight = DT_LIB_HISTOGRAM_HIGHLIGHT_GREEN;
-      gtk_widget_set_tooltip_text(widget, d->red ? _("click to hide green channel")
+      dt_control_change_cursor(GDK_LEFT_PTR);
```

Comment:
if we add dt_control_change_cursor() here we probably want to add it for the red case too.

But is has been removed in 31539181912256535963031360d864be38955f99 2 days ago. I tested this and it was ok without those call to change the cursor. The fix is probably to not reintroduce them.

Reply:
Ok, didn't trigger the "resolve conflicts" flag this time, so I think it should be ok.
Dramas 'cos I couldn't download from github :-(